2019-December.html

```diff
@@ -0,0 +1,247 @@
+                    <div class="fragment grow hoster-hiring">
+                        We're hiring!
+                    </div>
+                </section>
+
+                <!-- <section data-background="img/pagination_promo.jpg">
+                </section> -->
+
+                <section>
+                    <h1>Tonight's Speaker</h1>
+
+                    <div class="two-columns" style="margin-top: 4em;">
+                        <div>
+                            <img id="speaker-face" class="speaker" src="" alt="">
+                        </div>
+
+                        <div>
+                            <h3 id="speaker-name" class="fancy"></h3>
+                        </div>
+                    </div>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Conferences and Events</h1>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">PHP Benelux</h1>
+                    <img width="600" src="img/conference-logos/phpbnl-2019.png">
+                    <h4>January 24th - 25th</h4>
+                    <h4>Antwerp, Belgium</h4>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">DDD Europe</h1>
+                    <img width="600" src="img/conference-logos/ddd-europe-2019.png">
+                    <h4>February 3rd - 7th</h4>
+                    <h4>Amsterdam, The Netherlands</h4>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">PHP UK</h1>
+                    <img width="600" src="img/conference-logos/php-uk-2015.png">
+                    <h4>February 19th - 20th</h4>
+                    <h4>City of London, UK</h4>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">$Loop->Run();</h1>
+                    <img width="600" src="img/conference-logos/loop-run-2020.png">
+                    <h4>March 6th-7th</h4>
+                    <h4>Barcelona, Spain</h4>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">PHP Yorkshire</h1>
+                    <img width="600" src="img/conference-logos/phpyorkshire-2017.png">
+                    <h4>April 3rd - 4th</h4>
+                    <h4>York, UK</h4>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">PHP Serbia</h1>
+                    <img width="600" src="img/conference-logos/phpserbia-2020.png">
+                    <h4>May 21th-23th</h4>
+                    <h4>Belgrade, Serbia</h4>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">PHP Day</h1>
+                    <img width="600" src="img/conference-logos/phpday-2017.svg">
+                    <h4>May 14th-15th</h4>
+                    <h4>Verona, Italy</h4>
+                </section>
```

Comment:
Please swap these two to keep things in chronological order :grimacing:

Reply:
Done